_includes/default-header.html

```diff
@@ -1,30 +1,41 @@
-<nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
-  <div class="container">
-    <div class="navbar-brand">
-      
-      <a class="navbar-item" href="{{ site.baseurl }}/">
-          <img src="{{ site.baseurl }}/images/logo.png" alt="DNAnalyzer">
+<nav class="navbar" role="navigation" aria-label="main navigation">
+  <div class="navbar-brand">
+      <a class="navbar-item logo" href="{{ site.baseurl }}/">
+          DNAnalyzer
       </a>
-
-      <a
-        role="button"
-        class="navbar-burger burger"
-        aria-label="menu"
-        aria-expanded="false"
-        data-target="top-menu"
-      >
-        <span aria-hidden="true"></span> <span aria-hidden="true"></span>
-        <span aria-hidden="true"></span>
+      <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false"
+          data-target="navbarBasicExample">
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
       </a>
-    </div>
+  </div>
+
+  <div id="navbarBasicExample" class="navbar-menu">
+      <div class="navbar-start">
+          <a href="{{ site.baseurl }}/" class="navbar-item is-active">
+              Home
+          </a>
+
+          <a href="{{ site.baseurl }}/about" class="navbar-item">
+              About
+          </a>
+          <a class="navbar-item" href="https://streetscience.community/">
+              Street Science Community
+          </a>
+      </div>
 
-    <div id="top-menu" class="navbar-menu">
       <div class="navbar-end">
-        <a class="navbar-item" href="{{ site.baseurl }}/about"> About </a>
-        <a class="navbar-item" href="{{ site.url }}">
-			<img src="{{ site.baseurl }}/images/logo_streetscience.png" alt="Street Science Community">
-		</a>
+          <div class="navbar-item">
+              <div class="buttons">
+                  <a class="button login" href="{{ site.baseurl }}/game/intro/">
+                      Log in
+                  </a>
+                  <a class="button sign-up" href="{{ site.baseurl }}/game/intro/">
```

Comment:
It would be great if after clicking on 'log in' and 'sign up' buttons the users would be directed to a placeholder page saying: 'coming soon'.
At the moment they are directed to the start of the game.

Reply:
I added a coming_soon page and linked all the necessary buttons to it, please have a look!